iphone/cli/commands/_build.js

```diff
@@ -7102,6 +7102,8 @@ iOSBuilder.prototype.invokeXcodeBuild = async function invokeXcodeBuild(next) {
 			this.logger.warn(`The app is using native modules (${Array.from(this.legacyModules)}) that do not support arm64 simulators, we will exclude arm64. This may fail if you're on an arm64 Apple Silicon device.`);
 			args.push('EXCLUDED_ARCHS=arm64');
 		}
+	} else if( this.target ==='device' || this.target ==='dist-adhoc' ){
```

Comment:
<!--
  6 failure:  iphone/cli/comman..., iphone/cli/comman..., iphone/cli/comman..., iphone/cli/comman..., iphone/cli/comman..., iphone/cli/comman...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: iphone/cli/commands/_build.js;
  Line: 7105;
-->
- :no_entry_sign: iphone/cli/commands/_build.js line 7105 – Expected space(s) after "if". (keyword-spacing)
- :no_entry_sign: iphone/cli/commands/_build.js line 7105 – Missing space before opening brace. (space-before-blocks)
- :no_entry_sign: iphone/cli/commands/_build.js line 7105 – Operator '===' must be spaced. (space-infix-ops)
- :no_entry_sign: iphone/cli/commands/_build.js line 7105 – Operator '===' must be spaced. (space-infix-ops)
- :no_entry_sign: iphone/cli/commands/_build.js line 7105 – There should be no space after this paren. (space-in-parens)
- :no_entry_sign: iphone/cli/commands/_build.js line 7105 – There should be no space before this paren. (space-in-parens)